package_manager/util_test.py

```diff
@@ -66,5 +66,13 @@ def test_generate_debian_os_release(self):
             util.generate_os_release(distro, output_file)
             self.assertEqual(expected_output, output_file.getvalue())
 
+    def test_build_os_release_tar(self):
+        util.build_os_release_tar("buster", "etc/os-release", "etc", "release.tar")
+        os.remove("etc/os-release")
+        os.rmdir("etc")
```

Comment:
This feels a bit strange to me. Anybody with better python skills got an idea on how to work with generated files? I've tried using temp files and folders; but the tar command is a bit picky on folder structures.